cve_bin_tool/vex_manager/generate.py

```diff
@@ -95,10 +98,10 @@ def generate_vex_filename(self) -> str:
         Returns:
             str: The generated VEX filename.
         """
-        now = datetime.now().strftime("%Y-%m-%d.%H-%M-%S")
         filename = os.path.abspath(
             os.path.join(
-                os.getcwd(), f"{self.product}_{self.release}_{self.vextype}.{now}.json"
+                os.getcwd(),
+                f"{self.product}_{self.release}_{self.vendor}_{self.vextype}.json",
```

Comment:
Can you change this to use Path instead of using os.path? I know you're just editing something that was there but since it's Path above we might as well make them match while we're at it.

Reply:
@terriko I have made changes locally but I am fix some minor bugs although could you take a look at the Docs PR and let me know are there anything that can be improved in that, Thanks!